daffodil-core/src/main/scala/org/apache/daffodil/dpath/Expression.scala

```diff
@@ -2177,7 +2180,27 @@ case class DFDLXShiftExpr(nameAsParsed: String, fnQName: RefQName,
     res
   }
 }
+case class DFDLXBitExpr(nameAsParsed: String, fnQName: RefQName,
+                        args: List[Expression], constructor: (List[CompiledDPath], NodeInfo.Kind) => RecipeOp)
+  extends FunctionCallBase(nameAsParsed, fnQName, args) {
+  override lazy val inherentType = {
+    val arg0Type = args(0).inherentType
+    val arg1Type = args(1).inherentType
+    val argInherentType = if (arg1Type.isSubtypeOf(arg0Type)) arg0Type else arg1Type
+    argInherentType
+  }
+    override def targetTypeForSubexpression(subexp: Expression): NodeInfo.Kind = inherentType
 
+  override def compiledDPath: CompiledDPath = {
+    checkArgCount(2)
```

Comment:
This and the next 2 codecov warnings make it clear you have no test (as of when codecov ran) that exercises this function, because this code must be exercised by any such test.